editor.html

```diff
@@ -27,55 +32,70 @@
-			<li><a onclick="chooseBlack()" class="black-text">Black</a></li>  
-			<li><a onclick="chooseRed()" class="red-text">Red</a></li>  
-			<li><a onclick="chooseGreen()" class="green-text">Green</a></li>  
-			<li><a onclick="choosePink()" class="pink-text">Pink</a></li>
-			<li><a onclick="chooseGolden()" class="golden-text">Golden</a></li> 
-	 	</ul>
+		
      <!--Nav 2-->
-     <nav class="nav2" id="navbar2">
-        <div class="nav-wrapper2">
-            <a href="#" data-target="mobile-demo" class="white-text sidenav-trigger"><i
-                class="material-icons">menu</i></a>
-            <ul class="right hide-on-med-and-down">
-                <li><a class="nav2-text dropdown-trigger" data-target="dropdown-ink" href="editor.html">Change Ink<i class="material-icons right">arrow_drop_down</i></a></li>
-                <li><a class="nav2-text dropdown-trigger" data-target="dropdown-font">Choose Font<i class="material-icons right">arrow_drop_down</i></a></li>
-                <li><a class="nav2-text dropdown-trigger" data-target="dropdown-page">Choose Page<i class="material-icons right">arrow_drop_down</i></a></li>
-				<li><a class="nav2-text" onclick="save(str(pageNum))">Download</a></li>
+     <nav class="navbar nav2" id="navbar2">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                  </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav nav-main">
+                <li class="nav-item dropdown"></li>
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle text-white main-text" href="#" id="navbarDropdown1" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Change Ink
+                    </a>
+                    <div class="dropdown-menu dropdown-content" aria-labelledby="navbarDropdown1">
+                            <a onclick="chooseBlue()" class="blue-text dropdown-item">Blue</a></li>  
+                            <a onclick="chooseBlack()" class="black-text dropdown-item">Black</a></li>  
+                            <a onclick="chooseRed()" class="red-text dropdown-item">Red</a></li>  
+                            <a onclick="chooseGreen()" class="green-text dropdown-item">Green</a></li>  
+                            <a onclick="choosePink()" class="pink-text dropdown-item">Pink</a></li>  
+                    </div>
+                </li>
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle text-white main-text" href="#" id="navbarDropdown2" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
refer to the documentation of official dropdown from  bootstrap https://getbootstrap.com/docs/4.5/getting-started/introduction/